src/commands/ext-install.ts

```diff
@@ -114,17 +116,49 @@ async function installExtension(options: InstallExtensionOptions): Promise<void>
 /**
  * Command for installing a extension
  */
-export default new Command("ext:install <extensionName>")
-  .description("install an extension, given <extensionName> or <extensionName@versionNumber>")
+export default new Command("ext:install [extensionName]")
+  .description(
+    "install an extension, provide [extensionName] or [extensionName@versionNumber] or omit to see all available extensions."
+  )
   .option("--params <paramsFile>", "name of params variables file with .env format.")
   .before(requirePermissions, ["firebasemods.instances.create"])
   .before(ensureExtensionsApiEnabled)
   .action(async (extensionName: string, options: any) => {
+    const projectId = getProjectId(options, false);
+    const paramFilePath = options.params;
+    let learnMore = false;
+    if (!extensionName) {
+      learnMore = true;
+      const officialExts = await getExtensionRegistry();
+      extensionName = await promptOnce({
+        name: "input",
+        type: "list",
+        message:
+          "Which official extension would you like to install?\n" +
+          "  Select an extension, and press Enter to learn more.",
```

Comment:
nit - "... , then press ..."

[and I really like that it says "... to learn more" b/c it signals that the installation won't immediately proceed when the user selects the extension from the list :-) ]

Reply:
Change, and thanks :)

Reply:
Changed and thanks :)